firestore-perspective-toxicity/README.md

```diff
@@ -0,0 +1,13 @@
+# Analyze Toxicity with Perspective API
+
+**Author**: Perspective API (**[https://perspectiveapi.com](https://perspectiveapi.com/)**)
```

Comment:
Do we want to say Jigsaw here? (Jigsaw is the organization, Perspective is the product name)

Reply:
SGTM -- would we still link to perspectiveapi.com?

Reply:
I think it's best to just say 'Jigsaw' and link to the Jigsaw site (https://jigsaw.google.com/)